src/main/scala/ai/privado/tagger/AssetTagger.scala

```diff
@@ -0,0 +1,33 @@
+package ai.privado.tagger
+
+import ai.privado.cache.RuleCache
+import ai.privado.model.{InternalTag, RuleInfo}
+import io.shiftleft.semanticcpg.language.*
+import io.shiftleft.codepropertygraph.generated.{Cpg, EdgeTypes, Languages, Operators}
+import ai.privado.utility.Utilities.storeForTag
+import io.shiftleft.codepropertygraph.generated.nodes.NewTag
+
+class AssetTagger(cpg: Cpg) extends PrivadoSimpleCpgPass(cpg) {
```

Comment:
Missing test cases.

Reply:
This is only for POC purpose, will be removed when POC is completed